api/tools/src/main/resources/data_dictionary_exports/AllOfUsDatasetV2.yaml

```diff
@@ -0,0 +1,6392 @@
+meta_data:
```

Comment:
Is there a strong reason to host this in our codebase, as opposed to static files in GCS? (Maybe GCS, unsure. There's several ways to host it probably)

Do we expect to be maintaining these yamls? Or have them handed to us?

Reply:
I'd especially worry as we roll out N cdr versions, we end up with N data dictionary export files here.

Reply:
+1. If we do need it in github, maybe a separate repo, but GCS is very easy in terms of setup, automation, and interactive use for non-programmers.  This is more content than code.

Thinking more broadly, if we could collect all the auxiliary information we need with each CDR release (including ds_linking and and other schema-ish knowledge) into one "place" or "category" apart from the main line of source control, that would be helpful.

Reply:
Nope. This was taken as the path of least resistance. I have filed https://precisionmedicineinitiative.atlassian.net/browse/RW-3705 as the follow up to address this.

Reply:
Sounds good. It's not hurting anything yet.